package.json

```diff
@@ -19,7 +19,7 @@
     "url": "https://github.com/reactioncommerce/reaction/issues"
   },
   "dependencies": {
-    "@babel/runtime": "7.0.0-beta.49",
+    "@babel/runtime": "^7.0.0-beta.49",
```

Comment:
Please remove this change. We purposefully fully pin packages for security reasons